x/concentrated-liquidity/incentives_test.go

```diff
@@ -183,25 +183,34 @@ func expectedIncentivesFromRate(denom string, rate sdk.Dec, timeElapsed time.Dur
 }
 
 // expectedIncentivesFromUptimeGrowth calculates the amount of incentives we expect to accrue based on uptime accumulator growth.
+// It only factors in accumulators corresponding to authorized uptimes
 //
 // Assumes `uptimeGrowths` represents the growths for all global uptime accums and only counts growth that `timeInPool` qualifies for
 // towards result. Takes in a multiplier parameter for further versatility in testing.
 //
 // Returns value as truncated sdk.Coins as the primary use of this helper is testing higher level incentives functions such as claiming.
-func expectedIncentivesFromUptimeGrowth(uptimeGrowths []sdk.DecCoins, positionShares sdk.Dec, timeInPool time.Duration, multiplier sdk.Int) sdk.Coins {
+func (s *KeeperTestSuite) expectedIncentivesFromUptimeGrowth(uptimeGrowths []sdk.DecCoins, positionShares sdk.Dec, timeInPool time.Duration, multiplier sdk.Int) sdk.Coins {
```

Comment:
Note to reviewer: essentially all incentives tests related to supported or authorized uptimes lean on the `expectedIncentivesFromUptimeGrowth` and `addToUptimeAccums` abstractions, so it is sufficient to update these two.